runtime/web/shell.html

```diff
@@ -160,11 +160,14 @@
 
   <script>
     // PWA
-    if (navigator.serviceWorker && !navigator.serviceWorker.controller) {
+    if (navigator.serviceWorker) {
+      if (navigator.serviceWorker.controller) {
```

Comment:
navigator.serviceWorker.controller means a registered service worker, so you missed a `!`, only when no controller should do register

Reply:
Thank you for pointing this, I uploaded an update to fix it